src/locales/en.json

```diff
@@ -121,6 +115,7 @@
     "usage_cost_title": "Usage cost"
   },
   "chart": {
+    "cost": "Cost",
```

Comment:
This is not a chart string.

"cost_label" is used in the dashboard cards, while "cost_column_title" is a table heading in details.

Reply:
Done